src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer/index.js

```diff
@@ -31,7 +31,11 @@ class PreRenderer extends React.Component {
         const node = this.ref.getScrollableNode();
         const horizontalOverflow = node.scrollWidth > node.offsetWidth;
 
-        if ((event.currentTarget === node) && horizontalOverflow) {
+        /* Scrolling horizontally with two fingers on the touchpads,
```

Comment:
I think this comment could be a little shorter/clearer. Maybe something like: 

```
// Account for vertical scrolling variation when horizontally scrolling via touchpad by checking a large delta. 
```

Reply:
> I think this comment could be a little shorter/clearer. Maybe something like:
> 
> ```
> // Account for vertical scrolling variation when horizontally scrolling via touchpad by checking a large delta. 
> ```

@NikkiWines comment refactored. PR is ready 👍